tuxemon/core/components/event/actions/player.py

```diff
@@ -254,41 +254,6 @@ def add_monster(self, game, action):
         game.player1.add_monster(current_monster)
 
 
-    def add_item(self, game, action):
```

Comment:
We'd actually still need an `add_item` method here in `core.components.event.actions.player` so that you'd have the ability to add items to the player's inventory from an event defined in a map file. Everything in `core.components.event` are functions that you can use in the [Tiled map editor](http://mapeditor.org) for in-game events. The actual logic for adding items to a player's inventory should definitely be moved into the `core.components.player.Player` class though. Once the logic has been moved, this method could just call the player's `add_item` method like this:

```python
player = game.player1
player.add_item(item_to_add)
```